src/Carter.Analyzers/Carter.Analyzers.csproj

```diff
@@ -0,0 +1,35 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
I'd like to ship the analyser with Carter so should Carter reference this project that is then embedded in Carter's nuget package?